### ISSUE TITLE
How can I detect inputpicker is initialized or not!

### ISSUE THREAD

**Issue by mahavirsinhpadhiyar** [ukalpa/inputpicker#36]
How can I check if inputpicker is initialized or not.
My requirement here is I have to bind the inputpicker again and again on change of some other html element but after destroy I am not able to reinitialized it again so I want to check that if it is already initialized then and then I will destroy it.

Is there any function I can use to determine that inputpicker is already applied or not!

**Comment by mahavirsinhpadhiyar** [comment]
Yes, it applies inputpicker-original class but when I destroy it, inputpicker-original class remains as it is.
How can I reset with blank values and if want to destroy how can I destroy whole and make input as it is before.

**Comment by ukalpa** [comment]
How do you destroy it? Please use “$('#test').inputpicker('destroy');“ to destroy if you want to make it as before

**Comment by mahavirsinhpadhiyar** [comment]
Yes I am destroying it using the same method “$('#test').inputpicker('destroy');“ but when I initialize again as per requirement it throws the 
![error_inputpicker_one](https://user-images.githubusercontent.com/26636419/50958394-9c5ec180-14e6-11e9-821d-579b77e8fbd8.PNG)
following error:

Uncaught TypeError: Cannot read property 'top' of undefined

**Comment by ukalpa** [comment]
Can you please confirm you version is the latest and tell me what is your operation system and web browser?
It’s too late now, I will have a look at it tomorrow

**Comment by mahavirsinhpadhiyar** [comment]
I have downloaded your js from [jqueryscript](https://www.jqueryscript.net/form/Multi-column-Dropdown-Plugin-jQuery-Inputpicker.html) site and no version information I found in the js.
I am using Windows 10 and using the latest chrome.

Please give me any quick solution as of now if possible.

**Comment by ukalpa** [comment]
You can add a special class when imitating a inputpick object yourself to check if it’s status as a temporary solution

**Comment by mahavirsinhpadhiyar** [comment]
No, it destroys successfully but I am not able to initialize again after destroying so any temporary  solution for that!

**Comment by mahavirsinhpadhiyar** [comment]
As of now without destroying it, I am reinitializing it again and that works fine now.
But still, I don't want to set as it is when there is no data. So in the case of blank data I want to destroy it.

So please look into the issue I raised and give some permanent solution, thanks.

And thanks for such a good and amazing js you have given to the community.

**Comment by ukalpa** [comment]
Hi @mahavirsinhpadhiyar ,
Can you please try "$('#e1').inputpicker('destroy')" to unmount the plugin for the input?

**Comment by mahavirsinhpadhiyar** [comment]
Yes I have tried this before I already mentioned in the previous [comments](https://github.com/ukalpa/inputpicker/issues/36#issuecomment-453024668).
